tools/gstswitchcontrollerintrospection.c

```diff
@@ -0,0 +1,115 @@
+/* GstSwitch
```

Comment:
This copyright header is wrong. It should be;
```c
/* gst-switch							    -*- c -*-
 * Copyright (C) 2015 Peter XXXX <xxx@xxxxx>
 *
 * This file is part of gst-switch.
 *
 * gst-switch is free software: you can redistribute it and/or modify it under
 * the terms of the GNU General Public License as published by the Free
 * Software Foundation, either version 3 of the License, or (at your option)
 * any later version.
 *
 * This program is distributed in the hope that it will be useful, but WITHOUT
 * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
 * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
 * more details.
 *
 * You should have received a copy of the GNU General Public License along with
 * this program.  If not, see <http://www.gnu.org/licenses/>.
 */
```